rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -249,24 +250,10 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
       }
     }
 
-  // TODO: Remove ignore when bug RCHAIN-3917 is fixed.
-  it should "be repeatable when generated" ignore {
+  "replay the same term with same event logs" should "get the same cost" in {
```

Comment:
We don't want to remove existing test, it's testing bigger scope of Rholang terms.

Please add this as an additional test which is testing much smaller scope.

Reply:
No tests are remove . Why did you say that? I just rephrased it.